sklearn/linear_model/ransac.py

```diff
@@ -269,7 +284,10 @@ def fit(self, X, y):
                 continue
 
             # fit model for current random sample set
-            base_estimator.fit(X_subset, y_subset)
+            if base_estimator_sample_weight is None:
+                base_estimator.fit(X_subset, y_subset)
+            else:
+                base_estimator.fit(X_subset, y_subset, sample_weight= base_estimator_sample_weight[subset_idxs])
```

Comment:
Can you elaborate further? Is it similar to the else statement here https://github.com/scikit-learn/scikit-learn/blob/master/sklearn/ensemble/bagging.py#L96 ?